docker/jenkins-images/Dockerfile-k8s-test-h2o

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:latest
+
+RUN apt-get update \
+	&& apt-get install default-jdk -y
```

Comment:
This is fine for now, but eventually we should be able to control what version of Java is used.